include/libtorrent/string_view.hpp

```diff
@@ -41,11 +41,13 @@ POSSIBILITY OF SUCH DAMAGE.
 #include <boost/utility/string_ref.hpp>
 namespace libtorrent {
 using string_view = boost::string_ref;
+using wstring_view = boost::wstring_ref;
```

Comment:
"important" here in a relative sense...I think it's important mostly for code consistency. Also, this type does not exist in the C++ standard (the future one).

btw, I think that all the Unicode related code should be configurable, but not exposed in ABI